ui/components/ui/actionable-message/actionable-message.js

```diff
@@ -82,6 +91,10 @@ ActionableMessage.propTypes = {
     label: PropTypes.string,
     onClick: PropTypes.func,
   }),
+  primaryActionV2: PropTypes.shape({
+    label: PropTypes.string,
+    onClick: PropTypes.func,
+  }),
```

Comment:
primaryActionV2 has been added to achieve this design:
![Uploading Screenshot 2021-11-19 at 5.32.17 AM.png…]()